### DISCUSSION TITLE
How to modify status code errors

### DISCUSSION THREAD

**Discussion by ShourovRoy** [strawberry-graphql/strawberry#2304] [Q&A]
Apollo Graphql has a function to modify status code on error that can be overridden which plays an important role for implementing refresh token. Is there any way to do that here?

**Comment by patrick91** [comment]
@ShourovRoy sorry for the late reply, no we don't support this. But it is probably a potential improvement for the future! :)

Reference from apollo server's doc: https://www.apollographql.com/docs/apollo-server/data/errors/#setting-http-status-code-and-headers
